### PR TITLE
msg/async/rdma: set correct value to memory manager

### PR DESCRIPTION
**Pull request by Adirl**
set the correct max_send_wr and max_recv_wr
print log messages accordingly

Signed-off-by: Adir Lev <adirl@mellanox.com>

### CODE REVIEW

**Comment by Adirl** [comment]
@yuyuyu101 pls review

File: src/msg/async/rdma/Infiniband.cc [hunk 1]
```diff
@@ -75,25 +75,26 @@ Infiniband::Infiniband(CephContext *cct, const std::string &device_name, uint8_t
   assert(NetHandler(cct).set_nonblock(device->ctxt->async_fd) == 0);
 
   max_recv_wr = device->device_attr->max_srq_wr;
-  if (max_recv_wr > cct->_conf->ms_async_rdma_receive_buffers) {
-    ldout(cct, 0) << __func__ << " max allowed receive buffers is " << max_recv_wr << " use this instead." << dendl;
+  if (max_recv_wr > cct->_conf->ms_async_rdma_receive_buffers){
```
Comment (yuyuyu101) [inline src/msg/async/rdma/Infiniband.cc]: format: space betwen ( {

File: src/msg/async/rdma/Infiniband.cc [hunk 2]
```diff
@@ -75,25 +75,26 @@ Infiniband::Infiniband(CephContext *cct, const std::string &device_name, uint8_t
...
     max_recv_wr = cct->_conf->ms_async_rdma_receive_buffers;
-  }
+    ldout(cct, 0) << __func__ << " assigning: " << max_recv_wr << " receive buffers" << dendl;
+  } else
+    ldout(cct, 0) << __func__ << " using the max allowed receive buffers: " << max_recv_wr << dendl;
+
   max_send_wr = device->device_attr->max_qp_wr;
-  if (max_send_wr > cct->_conf->ms_async_rdma_send_buffers) {
-    ldout(cct, 0) << __func__ << " max allowed send buffers is " << max_send_wr << " use this instead." << dendl;
+  if (max_send_wr > cct->_conf->ms_async_rdma_send_buffers){
```
Comment (yuyuyu101) [inline src/msg/async/rdma/Infiniband.cc]: nitto

**Comment by Adirl** [comment]
Done

File: src/msg/async/rdma/Infiniband.cc [hunk 3]
```diff
@@ -76,24 +76,25 @@ Infiniband::Infiniband(CephContext *cct, const std::string &device_name, uint8_t
...
+  } else
+    ldout(cct, 0) << __func__ << " using the max allowed receive buffers: " << max_recv_wr << dendl;
+
   max_send_wr = device->device_attr->max_qp_wr;
   if (max_send_wr > cct->_conf->ms_async_rdma_send_buffers) {
-    ldout(cct, 0) << __func__ << " max allowed send buffers is " << max_send_wr << " use this instead." << dendl;
     max_send_wr = cct->_conf->ms_async_rdma_send_buffers;
-  }
+    ldout(cct, 0) << __func__ << " assigning: " << max_send_wr << " send buffers"  << dendl;
+} else
```
Comment (yuyuyu101) [inline src/msg/async/rdma/Infiniband.cc]: "  } else { "

Comment (yuyuyu101) [inline src/msg/async/rdma/Infiniband.cc]: space indent

**Comment by Adirl** [comment]
Thanks for showing me the correct formating... will pass to my team as well.

**Pull request merged**